README.md

```diff
@@ -1,5 +1,10 @@
 # WakaTime CLI
 
+| branch  | build | coverage |
+| ---     | ---   | ---      |
+| develop | ![Build develop](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Create%20Release/develop) | [![Coverage Status](https://coveralls.io/repos/github/wakatime/wakatime-cli/badge.svg?branch=develop)](https://coveralls.io/github/wakatime/wakatime-cli?branch=develop)
```

Comment:
`Create Release` workflow actually does not represent the build process.

Reply:
What about having `Create Release` as **Unit Tests Status** and `Build and upload release assets` as **Build Status**?

Reply:
Yeah, this sounds like the right way :+1: